src/App.js

```diff
@@ -1,27 +1,26 @@
-import React, { Component } from 'react';
+import React, { Component } from "react";
 
 class App extends Component {
   render() {
     return (
       <div className="App">
-        <img 
-          src={require('./images/mentor.jpg')} 
-          alt="Mentor" 
-          className="App__background-image"
-        />
-        <header className="App__header">
-          <h1 className="App__title">Coding Coach</h1>
-          <h4 className="App__tagline">Connecting developers with mentors worldwide.</h4>
-          <button className="button button__white--fill">
-            <a 
-              href="https://docs.google.com/forms/d/1TraoygcJay39cLiBQM-ROlToy7xIgcdx_VvCQuE1lJw/edit#responses"
-              target="blank"
-              className="button__link"
-            >
-              Get Started
-            </a>
-          </button>
-        </header>
+        <section className="App__hero">
+          <header className="App__header">
+            <h1 className="App__title">Coding Coach</h1>
+            <h4 className="App__tagline">
+              Connecting developers with mentors worldwide.
+            </h4>
+            <button className="button button__white--fill">
```

Comment:
not a big deal, but probably should have a different name for this `button__white--fill`, if you're following BEM (as it seems). Both `white` and `fill` seem modifiers. What do you think?

Reply:
Not really sure but why are we using class names with double underscores?

Reply:
I agree white & fill are both modifiers.